src/program/services/scrapers/base.py

```
@@ -32,8 +32,15 @@ def _initialize(self) -> None:
             if self.validate():
                 self.initialized = True
                 logger.success(f"{self.__class__.__name__} scraper initialized")
-        except Exception:
-            pass
+            else:
+                logger.warning(
+                    f"{self.__class__.__name__} scraper validate() returned False; "
+                    f"scraper is enabled in settings but will not be used"
+                )
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)